reference-ril/reference-ril.c

```diff
@@ -3064,20 +3064,25 @@ onRequest (int request, void *data, size_t datalen, RIL_Token t)
                it will call GET_CURRENT_CALLS and determine success that way */
             RIL_onRequestComplete(t, RIL_E_SUCCESS, NULL, 0);
             break;
-        case RIL_REQUEST_SWITCH_WAITING_OR_HOLDING_AND_ACTIVE:
+        case RIL_REQUEST_SWITCH_WAITING_OR_HOLDING_AND_ACTIVE: {
             // 3GPP 22.030 6.5.5
             // "Places all active calls (if any exist) on hold and accepts
             //  the other (held or waiting) call."
-            at_send_command("AT+CHLD=2", NULL);
+            p_response = NULL;
```

Comment:
Why do we have to initialize it to NULL here? And could we do initialization in declaration statement if initialization is necessary? 
e.g. ```ATResponse *p_response = NULL;```